extensions/blocks/membership-button/view.js

```diff
@@ -0,0 +1,76 @@
+/* global tb_show, tb_remove */
+
+/**
+ * Internal dependencies
+ */
+import './view.scss';
+const name = 'membership-button';
+const blockClassName = 'wp-block-jetpack-' + name;
+
+/**
+ * Since "close" button is inside our checkout iframe, in order to close it, it has to pass a message to higher scope to close the modal.
+ *
+ * @param {event} eventFromIframe - message event that gets emmited in the checkout iframe.
+ * @listens message
+ */
+function handleIframeResult( eventFromIframe ) {
+	if ( eventFromIframe.origin === 'https://subscribe.wordpress.com' && eventFromIframe.data ) {
+		const data = JSON.parse( eventFromIframe.data );
+		if ( data && data.action === 'close' ) {
+			window.removeEventListener( 'message', handleIframeResult );
+			tb_remove();
+		}
+	}
+}
+
+function activateSubscription( block, blogId, planId, poweredText ) {
+	block.addEventListener( 'click', () => {
+		tb_show(
+			null,
+			'https://subscribe.wordpress.com/memberships/?blog=' +
```

Comment:
If my site is in French (or any other language supported by Stripe), could we pass an extra parameter to get a Stripe interface in French?

Reply:
Will do!

Reply:
Done!